src/connector/src/source/kafka/client_context.rs

```diff
@@ -144,9 +144,11 @@ impl KafkaContextCommon {
                         )
                         .await
                     })
-                })
-                .unwrap()
-                .unwrap()
+                });
+                result
+                    .map_err(|_e| "generating AWS MSK IAM token timeout".to_owned())?
```

Comment:
We should retain the original error if it provides time elapsed. It will be helpful for debugging.